pridepy/download/pride.py

```
@@ -0,0 +1,1046 @@
+        :param file_name: The file name to be downloaded
+        :param username: Username with access to the dataset
+        :param password: Password for user with access to the dataset
+        """
+
+        auth = Authentication()
+        auth_token = auth.get_token(username, password)
+        validate_token = auth.validate_token(auth_token)
+        logging.info("Valid token after login: {}".format(validate_token))
+
+        url = self.API_PRIVATE_URL + "/projects/{}/files?search={}".format(accession, file_name)
+        content = requests.get(url, headers={"Authorization": "Bearer {}".format(auth_token)})
```

Ruff (0.15.14)

[error] Probable use of `requests` call without timeout

(S113)